queue_services/entity-pay/src/entity_pay/worker.py

```diff
@@ -121,7 +121,7 @@ async def process_payment(payment_token, flask_app):
 
             try:
                 await publish_email_message(
-                    qsm, APP_CONFIG.EMAIL_PUBLISH_OPTIONS['subject'], filing_submission, 'filed')
+                    qsm, APP_CONFIG.EMAIL_PUBLISH_OPTIONS['subject'], filing_submission, filing_submission.status)
```

Comment:
We should move this to be before we publish to the filer queue (just in case the filer processes it super fast and then the status of the filing == completed before we publish the 'paid' email to the email queue). Not sure if that would ever happen, but I think its possible

Reply:
That is possible. Also if this message fails for some reason, when it gets reprocessed, it may send as complete. Do you think we should explicitly set the status to Filing.Status.PAID.value along with moving the code?

Reply:
Yes thats a great idea I think